Scoop/Controller/SearchViewController.swift

```diff
@@ -97,13 +102,11 @@ class SearchViewController: UIViewController {
         departureView.addSubview(departureImageView)
         
         departureTextField.placeholder = "departure date"
-        let datePicker = UIDatePicker()
+        
         datePicker.datePickerMode = .date
         datePicker.backgroundColor = .clear
-
-        departureView.addSubview(datePicker)
-        datePicker.addTarget(self, action: #selector(openDatePicker), for: .touchDown)
         departureView.addSubview(datePicker)
+        datePicker.addTarget(self, action: #selector(onDateValueChanged(_:)), for: UIControl.Event.valueChanged)
```

Comment:
You can just write `.valueChanged`

Reply:
Is this the same thing for other .addTarget setups in this view? I saw:

`arrivalTextField.addTarget(self, action: #selector(arrivalSearch), for: UIControl.Event.touchDown)`

Should I also change that to just `.touchDown` as well?